### DISCUSSION TITLE
Is the strategy option necessary?

### DISCUSSION THREAD

**Discussion by diegohaz** [floating-ui/floating-ui#1813] [Q&A] [answered]
I noticed that if I set `position: fixed` to the floating element CSS before the `computePosition()` call, everything seems to work fine, which makes me wonder if setting `strategy: "fixed"` is necessary.

**Comment by atomiks** [comment, marked as answer]
It is, what may not be necessary is what the value is prior to the call, but the call itself has a ton of checks which changes the coordinates that are returned. In some cases it doesn't matter though, it depends on the DOM layout.
